ooniprobe/Test/Suite/AbstractSuite.h

```diff
@@ -13,6 +13,7 @@
 @property NSMutableArray *testList;
 @property int measurementIdx;
 @property (nonatomic) UIBackgroundTaskIdentifier backgroundTask;
+@property BOOL interrupted;
```

Comment:
I assume the default is `atomic`? Maybe, can you make it explicitly `atomic`?

Reply:
There is no need to, unless we want `weak`